articles/sts_storage_locations.md

```diff
@@ -0,0 +1,172 @@
+---
+title: "STS Storage Locations"
+layout: article
+excerpt: Follow these steps to set up STS Storage Locations and use them to access your S3 data directly.
+category: howto
+---
+
+Using AWS Security Token Service (STS), Synapse can securely grant you temporary AWS credentials with access to data directly in S3. This can be useful if you want to:
+
+* Transfer data in bulk
+* Point your compute cluster directly at the data
```

Comment:
instead of 'directly at' one might say 'Allow your compute cluster to read S3 objects using the S3 APIs